subsys/bluetooth/services/fast_pair/fp_storage/fp_storage_ak.c

```diff
@@ -341,7 +341,7 @@ static void fp_storage_ak_empty_init(void)
 	atomic_set(&settings_loaded, true);
 }
 
-static int ak_delete(uint8_t index)
+int fp_storage_ak_delete(uint8_t index)
```

Comment:
I don't think it's very much needed. We have `_priv` headers already.

Reply:
I think it's definitely not a high-priority issue, but it could slightly improve code quality (we should keep in mind to always use `static` for function implementations that are not part of the API).